### PR TITLE
HCK-8896: Code improvements 

### PR DESCRIPTION
**Pull request by Nightlngale**
<table>
<td>
  <a href="https://hackolade.atlassian.net/browse/HCK-8896" title="HCK-8896" target="_blank">Sub-taskHCK-8896</a>  Finish refactoring
  </td></table>
  <br />
 

## Content

- Remove manual formatting
- Move the logic to the provider
- Remove unnecessary functions
- Remove duplicated () if the user’s input already contains them

### CODE REVIEW

File: forward_engineering/ddlProvider/ddlProvider.js [hunk 1]
```diff
@@ -1,13 +1,76 @@
...
+	getEntityProperties,
+	wrapInBrackets,
+} = require('../utils/general');
+const { getViewTagsStatement } = require('../helpers/unityTagsHelper');
+const { getTablePropertiesClause } = require('../helpers/tableHelper');
+const viewHelper = require('../helpers/viewHelper');
 
 module.exports = app => {
 	const { assignTemplates } = app.require('@hackolade/ddl-fe-utils');
+	const _ = app.require('lodash');
```
Comment (bigorn0) [inline forward_engineering/ddlProvider/ddlProvider.js:18]: @Nightlngale ideally getting deps like this should be migrated to direct deps in the plugin when possible :)

Comment (Nightlngale) [reply forward_engineering/ddlProvider/ddlProvider.js:18]: @bigorn0 I couldn’t agree more. But here I was trying to touch a small part of code related only to "Create view" script

Comment (bigorn0) [reply forward_engineering/ddlProvider/ddlProvider.js:18]: @Nightlngale that's true in general that we want to limit the impacted surface but for such tech debt and given how often we go into plugins code I think the opposite:  If we don't do our sanity/cleanup changes along the way when we are adapting plugins then we NEVER get the chance to actually tackle such debt :(

Comment (VitaliiBedletskyi) [reply forward_engineering/ddlProvider/ddlProvider.js:18]: I agree with Ugo it's a good opportunity to fix it. But I would propose fixing it in a separate PR. @Nightlngale could you please create PR with this tech improvement?

Comment (Nightlngale) [reply forward_engineering/ddlProvider/ddlProvider.js:18]: @bigorn0 @VitaliiBedletskyi yes, I will add this improvement in scope of the separate PR

**Review by VitaliiBedletskyi — APPROVED** [review]

**Pull request merged**